src/theme/style.scss

```diff
@@ -20,6 +20,11 @@ body {
   margin: 0;
   padding: 0;
   height: 100%;
+  min-height: 100vh;
+  background-image: url('/static/assets/ui/QuizBackground.png');
```

Comment:
It's better to not use absolute paths. Please change it to relative path (starting with './')